### PR TITLE
added max_sum code with readme updated

### PR DESCRIPTION
**Pull request by shraddhavp**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

added the code for a maximum sum in python

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

issue #937

### CODE REVIEW

**Comment by shraddhavp** [comment]
please review  and let me know if any changes @iamrajiv @Kajol-Kumari

File: Python/dp/max_sum.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+if __name__ == '__main__':
+    rectangle = [[2,  1, -3, -4,  5],
+                 [0,  6,  3,  4,  1],
+                 [2, -2, -1,  4, -5],
+                 [-3,  3,  1,  0,  3]]
+
+    result = max_sub_sub_rectangle(rectangle)
+    assert 18 == result.maxSum
+    print (result)
```
Comment (iamrajiv) [inline Python/dp/max_sum.py:57]: @shraddhavp Add time and space complexity.

File: Python/dp/README.md [hunk 2]
```diff
@@ -1 +1 @@
-a
+[Maximum sum rectangle of 2D array](dp/max_sum.py)
```
Comment (iamrajiv) [inline Python/dp/README.md:1]: @shraddhavp Do not add link of code to `Python/dp/README.md` instead add to `Python/README.md` in  `Competitive Programming` section.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@shraddhavp Remove unnecessary files like `ShellSort.cpp `. There should be only 2 files changed i.e. README and program but I can see there is 4 file changed please take a look. Format your code according to PEP 8 rules. You can refer to this http://pep8online.com/.

**Pull request closed without merging**